File: BackEnd/CasottoIdS/src/main/java/it/unicam/cs/CasottoIdS/models/Attrezzatura.java
package it.unicam.cs.CasottoIdS.models;

import org.springframework.data.annotation.Id;

public class Attrezzatura {

    @Id
    public String nome;
    private int quantita;

    public Attrezzatura(String nome, int quantita) {
        this.nome = nome;
        this.quantita = quantita;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getQuantita() {
        return quantita;
    }

    public void setQuantita(int quantita) {
        this.quantita = quantita;
    }
}

File: FrontEnd/Casotto/lib/models/Attrezzatura.dart


class Attrezzatura{

  late String nome;
  late int quantita;

  Attrezzatura(String nome, int quantita){
    this.nome = nome;
    this.quantita = quantita;
  }

  String getNome(){
    return nome;
  }

  void setNome(String nome){
  this.nome = nome;
  }

  int getQuantita(){
    return quantita;
  }

  void setQuanita(int quantita){
    this.quantita = quantita;
  }

  String toString() {
    return "Attrezzatura{" +
        "Nome=" +
        nome +
        ", quantità= " +
        quantita.toString() +
        " }";

  }

}